core/src/main/java/com/cognifide/aem/stubs/core/script/ConfigurableStubScriptManager.java

```diff
@@ -219,8 +266,11 @@ public String toString() {
     @AttributeDefinition(name = "Root Path")
     String resource_paths() default "/var/stubs";
 
-    @AttributeDefinition(name = "Extension")
-    String extension() default ".groovy";
+    @AttributeDefinition(name = "Script Extension")
+    String scriptExtension() default ".groovy";
+
+    @AttributeDefinition(name = "Mappings Extension")
```

Comment:
Singular form

Reply:
The extensions are configurable however the subfolders 'mappings' and 'scripts' are hardcoded

Reply:
done